template/azurePipelines/manifests/draft.yml

```diff
@@ -0,0 +1,30 @@
+variables:
+  - name: "PIPELINE_NAME"

```

Comment:
same here, we can remove separator

Reply:
I've removed the separators from the var names